src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -392,6 +515,13 @@ export type MarkdownEditorProps = {
   text: string;
 
   onEditText?: (text: string) => void;
+
+  /**
+   * List of users who have annotated current document and belong to active group.
+   * This is used to populate the @mentions suggestions, when `atMentionsEnabled`
+   * is `true`.
+   */
+  usersWhoAnnotated: UserItem[];
```

Comment:
Perhaps `usersWhoAnnotated` is too specific, considering we'll be replacing it with a list of group members in the next phase.